app/views/partials/channels-tab-routes.html

```diff
@@ -12,10 +12,19 @@
       <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
     </div>
     <div class="form-group" style="float: left; margin-left: 20px;">
-      <label>Primary Route</label><br />
-      <div class="btn-group" style="clear: both;">
-        <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="true">True</label>
-        <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="false">False</label>
+      <div style="float: left;">
+        <label>Primary Route</label><br />
+        <div class="btn-group" style="clear: both;">
+          <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="true">True</label>
+          <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="false">False</label>
+        </div>
+      </div>
+      <div style="float: left; margin-left: 20px;">
```

Comment:
I'm not a fan of inline styles, but I see they are everywhere in this page. The float you could replace with a pull-left bootstrap calls. I'm not sure the best way to do the spacing though.

[non-blocking PR comment]